cicd/bootstrap.sh

```diff
@@ -81,5 +81,13 @@ add_cicd_bin_to_path() {
 
 add_cicd_bin_to_path
 
-# log in to ephemeral cluster
-oc_wrapper login --token=$OC_LOGIN_TOKEN --server=$OC_LOGIN_SERVER
+# !!Hotswap based on availability
+status = $(curl -s -w 200 $OC_LOGIN_SERVER)
```

Comment:
can we wrap this in a function to "get available server" and then be explicit about which server we're connecting to ? 

This way we can also handle better the event of "no cluster being available" too

Reply:
added - be mindful, this is just for the pr_checks - mostly not for humans :)